src/github.com/getlantern/tarfs/demo/resources.go

```diff
@@ -0,0 +1,3 @@
+package main
```

Comment:
This file is not actual editable source code, not sure if we should add it to the source tree, we may as well generate it at build time.